pallets/passkey/src/lib.rs

```diff
@@ -40,6 +47,8 @@ mod tests;
 pub mod weights;
 pub use weights::*;
 
+#[cfg(feature = "runtime-benchmarks")]
+use frame_support::traits::{Currency, ReservableCurrency};
```

Comment:
I believe the Currency trait is also being deprecated in favor of the [Fungible](https://paritytech.github.io/polkadot-sdk/master/frame_support/traits/tokens/fungible/index.html) trait.